### PR TITLE
[Docs] - Testing Booster applications

### PR DESCRIPTION
**Pull request by juanjoman**
## Description
These are some guidelines that might be useful for people testing a Booster application. It helps also fixing some of the errors from https://github.com/boostercloud/booster/issues/627

## Changes
Add docs about how to test a Booster application

## Checks
- [X] Project Builds
- [X] Project passes tests and checks
- [X] Updated documentation accordingly

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -2957,6 +2958,76 @@ Variables
...
+    register.events(new MyEvent(command.ID, readModelName))
+  }
+}
+```
+
+#### Recommended files
+These are some files that might help you speed up your testing with Booster.
+
+```typescript
+// <root_dir>/test/expect.ts
```
Comment (charlietfe) [inline docs/README.md:3007]: Are we providing this? Where the expect comes from?

Comment (juanjoman) [reply docs/README.md:3022]: No we are not, this is a way of allowing the user to use more specific methods from `sinon-chai` and `chai-as-promised`. It's not mandatory but very useful for testing :v: @charlietfe

**Comment by davidverdu** [comment]
This is only the doc, isn't it? We will probably need to create another PR to add the pending mocha files. It would be awesome that our cli generate the test file as well as the src file when you run, for example `boost new:command CreateComment`. I'll create a ticket for the latter.

**Comment by juanjoman** [comment]
Yes @davidverdu this is only docs regarding this issue #627, until we fix it properly with the CLI

**Review by davidverdu — APPROVED** [review]

File: docs/README.md [hunk 2]
```diff
@@ -2957,6 +2958,91 @@ Variables
...
+"scripts": {
+  "test": "nyc --extension .ts mocha --forbid-only \"test/**/*.test.ts\""
+}
+```
+
+The only thing that you should add to this line are the `AWS_SDK_LOAD_CONFIG=true` and `BOOSTER_ENV=test` environment variables, so the script will look like this:
+
+```typescript
+"scripts": {
+  "test": "AWS_SDK_LOAD_CONFIG=true BOOSTER_ENV=test nyc --extension .ts mocha --forbid-only \"test/**/*.test.ts\""
```
Comment (adriangmweb) [inline docs/README.md:2975]: Why these env vars have to be added manually?

Comment (juanjoman) [reply docs/README.md:2975]: Each one solves a different "error":
- BOOSTER_ENV=test => `No environment found for your Booster application`
- AWS_SDK_LOAD_CONFIG=true => `region not found`

I'm not 100% sure why this error happens, but it's just a quick fix @adriangmweb

**Review by adriangmweb — APPROVED** [review]

**Pull request merged**